src/swap/sideshift.js

```diff
@@ -134,6 +131,22 @@ async function checkRateForError(
       request.toCurrencyCode
     )
   }
+
```

Comment:
Please clean this up with (nested) ES6 destructuring; it will make it a *lot* more readable as to what's going on. (less code to read)

Give it a shot, and let me know if you need help with it. 🙂